_events/sr2020/competition.md

```diff
@@ -14,9 +14,7 @@ Keep an eye on this page for updates as more information becomes available.
 
 ## Schedule
 
-The competition will be held from *Saturday 18th April* until *Sunday 19th April*.
```

Comment:
Maybe replace this with a comment about roughly when we hope to have a confirmed date and/or just saying that the date is yet to be confirmed.